### DISCUSSION TITLE
Start script after Phoniebox loads?

### DISCUSSION THREAD

**Discussion by alanrodriguezdotme** [MiczFlor/RPi-Jukebox-RFID#1222] [Q&A]
Hi everyone. I'm working on an update to my current Phoniebox. I'm adding an [Adafruit Neotrellis](https://www.adafruit.com/product/3954) to this new version, replacing the GPIO buttons I'm currently using. I'm a bit of a Python noob, so excuse me if this is a basic question: I have a script that makes the Neotrellis work properly, but I currently need to SSH into the Pi and run it manually. Is there a way to run the script after Phoniebox loads? 

I'm using a 2019 version of the project, if that helps at all.

I appreciate any help. 🙏

**Comment by s-martin** [comment]
Hi,

you would need to add a so called service to automatically load on boot.

example for a service file here: https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/misc/sampleconfigs/phoniebox-gpio-control.service.sample

You need to place the file here (you need sudo rights) `  /etc/systemd/system/`

and activate via

`systemctl start phoniebox-gpio-control.service` (Change to your filename)
`systemctl enable phoniebox-gpio-control.service`

**Reply by alanrodriguezdotme** [reply]
Yes, I'm semi-familiar with service. Currently using it for a 2x16 LCD display. Which is actually breaking my script all of a sudden. 

Thanks for the suggestion.
